jplag/src/main/java/de/jplag/GreedyStringTiling.java

```diff
@@ -97,7 +97,7 @@ public final JPlagComparison compareWithBaseCode(Submission firstSubmission, Sub
         return swapAndCompare(firstSubmission, secondSubmission, true);
     }
 
-    private final JPlagComparison swapAndCompare(Submission firstSubmission, Submission secondSubmission, boolean isBaseCodeComparison) {
+    private JPlagComparison swapAndCompare(Submission firstSubmission, Submission secondSubmission, boolean isBaseCodeComparison) {
```

Comment:
Why remove final here?

Reply:
private methods are not inherited therefore not final :)

Reply:
I missed the private here. I thought there was more intent here. Maybe I should not do code reviews during meetings! 😄